crates/bevy_core_pipeline/src/blit/mod.rs

```diff
@@ -36,28 +43,16 @@ impl FromWorld for BlitPipeline {
     fn from_world(render_world: &mut World) -> Self {
         let render_device = render_world.resource::<RenderDevice>();
 
-        let texture_bind_group =
-            render_device.create_bind_group_layout(&BindGroupLayoutDescriptor {
-                label: Some("blit_bind_group_layout"),
-                entries: &[
-                    BindGroupLayoutEntry {
-                        binding: 0,
-                        visibility: ShaderStages::FRAGMENT,
-                        ty: BindingType::Texture {
-                            sample_type: TextureSampleType::Float { filterable: false },
-                            view_dimension: TextureViewDimension::D2,
-                            multisampled: false,
-                        },
-                        count: None,
-                    },
-                    BindGroupLayoutEntry {
-                        binding: 1,
-                        visibility: ShaderStages::FRAGMENT,
-                        ty: BindingType::Sampler(SamplerBindingType::NonFiltering),
-                        count: None,
-                    },
-                ],
-            });
+        let texture_bind_group = render_device.create_bind_group_layout(
+            "blit_bind_group_layout",
+            &BindGroupLayoutEntries::sequential(
+                ShaderStages::FRAGMENT,
+                (
+                    texture_2d(TextureSampleType::Float { filterable: false }),
```

Comment:
i see you've chosen not to use the shorter methods (`texture_2d_f32(false)` here e.g.), was that deliberate or because they were added later?

Reply:
It's mostly because I'm not sure which to use. I like `texture_2d_f32(false)` better, but I don't like having a bare boolean parameter like that. It's very not obvious what it means. I was mostly waiting on getting everyone's opinion.